packages/core/src/components/button/abstractButton.tsx

```diff
@@ -29,6 +29,12 @@ export interface IButtonProps extends IActionProps {
     loading?: boolean;
 
     /**
+     * If set to `true`, the button will display in an active state. This is equivalent to setting `pt-active` via className.
```

Comment:
this line is too long, fails linting. please wrap after the period.

also, how about "Whether the button will..." ?